frontend/src/axios-instance.ts

```diff
@@ -1,15 +1,55 @@
 import axios from 'axios';
 
+function getCookieValue(key: String) {
+  const cookies = document.cookie.split(';');
+  for (let i = 0; i < cookies.length; i++) {
+    const cookie = cookies[i].trim();
+    if (cookie.startsWith(key + '=')) {
+      return cookie.substring(key.length + 1);
+    }
+  }
+  return null;
+}
+
+async function refreshAccessToken() {
+  // Call the API to refresh the access token using the refresh token
+
+  const response = await axios.post('/auth/token/refresh/', {
+    refresh: getCookieValue('refresh'),
+  });
+  // Save the new access token in the cookie
+  const accessToken = response.data.access;
+  // Set the expiration time to be 5 minutes
+  const expirationDate = new Date(Date.now() + 300 * 1000);
+  document.cookie = `token=${accessToken}`;
+  document.cookie = `expiration=${expirationDate.toUTCString()}`;
```

Comment:
just curious, it works with = in different lines?

Reply:
Yes, from testing both values are updated